packages/core/src/components/modal/modal.scss

```diff
@@ -0,0 +1,79 @@
+.modal {
+    position: absolute;
+    background: var(--bg-color);
+    z-index: 1000;
+    left: 50%;
+    top: 50%;
+    width: 50%; /* 6 cols = 50% */
+    transform: translate(-50%, -50%);
+    border-radius: 4px;
+    padding: 32px;
+}
+
+.modal-root {
+    display: block;
+}
+
+.modal__open-true {
+    display: grid;
+    grid-template-columns: repeat(12, 1fr); 
+    visibility: visible;
+}
+
+.modal__open-false {
+    display: none;
+    visibility: hidden;
```

Comment:
`display: none` => `visibility: hidden`. We probably don't need this.

Reply:
I think I used the display option just to mimic those used from Material UI and Bootstrap. This and had some odd glitch with visibility